settings.gradle

```diff
@@ -0,0 +1,36 @@
+rootProject.name = 'EssentialsXParent'
+
+include(':EssentialsX')
+project(':EssentialsX').projectDir = file('Essentials')
+
+// Modules
+include(':EssentialsXAntiBuild')
+project(':EssentialsXAntiBuild').projectDir = file('EssentialsAntiBuild')
+
+include(':EssentialsXChat')
+project(':EssentialsXChat').projectDir = file('EssentialsChat')
+
+include(':EssentialsXGeoIP')
+project(':EssentialsXGeoIP').projectDir = file('EssentialsGeoIP')
+
+include(':EssentialsXProtect')
+project(':EssentialsXProtect').projectDir = file('EssentialsProtect')
+
+include(':EssentialsXSpawn')
+project(':EssentialsXSpawn').projectDir = file('EssentialsSpawn')
+
+include(':EssentialsXXMPP')
+project(':EssentialsXXMPP').projectDir = file('EssentialsXMPP')
+
+// Providers
+include(':BaseProviders')
```

Comment:
maybe skip the custom project directory, just do `include(":providers:BaseProviders")` and update the project deps to match?